request/graphql/schema/generate.go

```diff
@@ -98,8 +98,24 @@ func (g *Generator) generate(ctx context.Context, collections []client.Collectio
 		if err != nil {
 			return nil, err
 		}
-		queryType.AddFieldConfig(f.Name, f)
 		generatedQueryFields = append(generatedQueryFields, f)
+
+		var isEmbedded bool
+		for _, definition := range collections {
+			if t.Name() == definition.Schema.Name && definition.Description.Name == "" {
+				isEmbedded = true
```

Comment:
question: You rather use this method than to use `continue` with a loop tag? The block content is short enough that if would still read easily. Fine either way. Just curious.

Reply:
I see loop tags as a bit unusual and my brain has not get chosen to either ban them or always use them.  Sometimes I use them sometimes I don't, *I think* I mostly only use them in tiny (~10 line or less) functions where it is hard for them to get lost.